specification.md

```diff
@@ -303,7 +311,7 @@ Content-Type: application/json
   ],
   "api_version": "1.0",
   "status_callback_urls": [
-    "https://examplecontroller.com/opengdpr_callbacks"
+    "https://examplecontroller.com/opendsr_callbacks"
```

Comment:
Maybe we should do the same as with `/requests` here:`https://exampleopendsrcontroller.com/callback` ?

Reply:
Agreed. This is cleaner, though controllers are free to set it to anything they like. Will update